Source/ZoomNet/Models/ChatMessageParty.cs

```diff
@@ -0,0 +1,31 @@
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Chat message party (i.e. sender or recipient) information.
+	/// </summary>
+	public class ChatMessageParty
```

Comment:
Is this class intended to be used exclusively for webhook content parsing? If so, it should be under the `ZoomNet.Models.Webhooks` namespace.

However, it's perfectly fine to leave this class in the `ZoomNet.Models` namespace if you think we can use this class to parse data returned from the API.